cfme/fixtures/generic_object.py

```diff
@@ -0,0 +1,65 @@
+import fauxfactory
+import pytest
+
+from cfme.services.myservice import MyService
+from cfme.utils.appliance import ViaREST
+from cfme.utils.rest import assert_response
+
+
+@pytest.fixture(scope="module")
+def gen_rest_service(appliance):
+    """Simple service create with rest for generic object association"""
+
+    rest_service = appliance.rest_api.collections.services.action.create(
+        name=fauxfactory.gen_numeric_string(16, start="gen_rest_serv", separator="-"), display=True
+    )
+    rest_service = rest_service[0]
+    yield rest_service
+    rest_service.action.delete()
+
+
+@pytest.fixture(scope="module")
+def gen_definition(appliance):
```

Comment:
Since you use `generic` below, I would expect this to be named `generic_definition`

Reply:
done